Header header

float32[] x_setSeqBase
float32[] y_setSeqBase

float32[] x_setSeqTrans
float32[] y_setSeqTrans

#transformation---------------------------------------------------------------------

float32[] transf #transVec